appveyor.yml

```diff
@@ -101,5 +101,5 @@ test_script:
     - cmd: mkdir conda_packages
     # Uncomment the following two lines to make any conda packages created
     # available as build artifacts in AppVeyor
-    #- cmd: 'copy /Y C:\Miniconda\conda-bld\win-32\*.bz2 conda_packages || cmd /c "exit /b 0"'
-    #- cmd: 'copy /Y C:\Miniconda-x64\conda-bld\win-64\*.bz2 conda_packages || cmd /c "exit /b 0"'
```

Comment:
No problem removing it. I find it anyway pretty useful to test more extensively the package that got created :)

To avoid polluting with other build I will push it when I know what to do with the "patch problem". Do you have any update on what should be done there?